### ISSUE TITLE
Android Studio 4.0 Alpha 7 Crash 

### ISSUE THREAD

**Issue by wezley98** [Triple-T/gradle-play-publisher#757]
### Describe the bug
Build Crashes

### How To Reproduce
1. Upgrade project to Android Studio 4.0 Alpha 7
2. Run task publishReleaseBundle

#### Versions
- Gradle Play Publisher: 2.6.1
- Gradle Wrapper: Gradle 6.1-rc-1
- Android Gradle Plugin: com.android.tools.build:gradle:4.0.0-alpha07

#### Tasks executed

publishReleaseBundle

### Expected behavior

Build completes

### Additional context (if a crash, provide stack trace)

```
Execution failed for task ':app:generateReleasePlayResources'.
> A failure occurred while executing com.github.triplet.gradle.play.tasks.GenerateResources$Generator
   > Could not create an instance of type com.github.triplet.gradle.play.tasks.GenerateResources$Generator.
      > Unable to determine constructor argument #1: missing parameter of type ProjectLayout, or no service of type ProjectLayout.
```

**Comment by SUPERCILEX** [comment]
Yup, already fixed. Use GPP v2.6.2.

**Comment by wezley98** [comment]
Fixed in v2.6.2

**Comment by wezley98** [comment]
Thanks just seen the release now, thanks for the fast fix 👍
